docs/config.yaml

```
@@ -101,6 +101,10 @@ streams:
       # OPTIONAL: allow building using OSBuild for artifacts where OSBuild is
       # not yet the default in CoreOS Assembler.
       osbuild_experimental: true
+      # OPTIONAL: stream will be set as "scheduled"
+      # If using, `type: mechanical` must also be set on the stream
+      # This is used for some RHCOS streams to be built every day by the build-mechanical job
```

Maybe we should document the semantics here. E.g.

```suggestion
      # OPTIONAL: stream will be set as "scheduled"
      # By default, `build-mechanical` builds all mechanical streams. When mechanical
      # streams have `scheduled` on, then `build-mechanical` will only build those streams.
```

?

---

WFM!